examples/directivities/simulation_with_measured_directivity.py

```diff
@@ -0,0 +1,187 @@
+
+In this example, we show how we can apply and use class DIRPATRir to open measured
+directivity files from the DIRPAT dataset.
+
+The created objects can be directly used to generate RIRs
+
+With DIRPATRir object we can generate RIRs with mics and source having either
+frequency independent CARDIOID patterns or
+freqeuncy dependent patterns from DIRPAT dataset.
+
+Parameters
+--------------------------------------
+    orientation :
+        class DirectionVector
+    path : (string)
+        Path towards the DIRPAT sofa file, the ending name of the file should be the same as specified in the DIRPAT dataset
+
+    DIRPAT_pattern_enum : (string)
+        Only used to choose the directivity patterns available in the specific files in the DIRPAT dataset
+
+    # AKG_c480_c414_CUBE.sofa DIRPAT file include mic patterns for CARDIOID ,FIGURE_EIGHT,HYPERCARDIOID ,OMNI,SUBCARDIOID
+    a)AKG_c480
+    b)AKG_c414K
+    c)AKG_c414N
+    d)AKG_c414S
+    e)AKG_c414A
+
+    Eigenmic directivity pattern file "EM32_Directivity.sofa", specify mic name at the end to retrive directivity pattern for that particular mic from the eigenmike
+    a)EM_32_* : where * \in [0,31]
+    For example EM_32_9 : Will retrive pattern of mic number "10" from the eigenmic.
+
+    # LSPs_HATS_GuitarCabinets_Akustikmessplatz.sofa DIRPAT file include source patterns
+    a)Genelec_8020
+    b)Lambda_labs_CX-1A
+    c)HATS_4128C
+    d)Tannoy_System_1200
+    e)Neumann_KH120A
+    f)Yamaha_DXR8
+    g)BM_1x12inch_driver_closed_cabinet
+    h)BM_1x12inch_driver_open_cabinet
+    i)BM_open_stacked_on_closed_withCrossoverNetwork
+    j)BM_open_stacked_on_closed_fullrange
+    k)Palmer_1x12inch
+    l)Vibrolux_2x10inch
+
+    fs : (int)
+        Sampling frequency of the filters for interpolation.
+        Should be same as the simulator frequency and less than 44100 kHz
+    no_points_on_fibo_sphere : (int)
+        Number of points on the interpolated Fibonacci sphere.
+        if "0" no interpolation will happen.
+
+
+This implementation shows how we can use both DIRPAT object and frequency independent directivity class CardioidFamily together.
+We can also use the objects separately.
+
+~ Prerak SRIVASTAVA, 8/11/2022
+"""
+
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy import signal
+from scipy.fft import fft, fftfreq
+from scipy.io import wavfile
+from scipy.signal import fftconvolve
+
+import pyroomacoustics as pra
+from pyroomacoustics.directivities import (
+    Cardioid,
+    DirectionVector,
+    FigureEight,
+    MeasuredDirectivityFile,
+    Rotation3D,
+)
+
+eigenmike = MeasuredDirectivityFile("EM32_Directivity", fs=16000)
+akg = MeasuredDirectivityFile("AKG_c480_c414_CUBE", fs=16000)
+
+rot_54_73 = Rotation3D([73, 54], "yz", degrees=True)
+
+dir_obj_Dmic = akg.get_mic_directivity("AKG_c414K", orientation=rot_54_73)
+dir_obj_Emic = eigenmike.get_mic_directivity("EM_32_9", orientation=rot_54_73)
+
+dir_obj_Cmic = FigureEight(
```

Comment:
Do you think it is good to comment here that the mic response is analytic for this case ?

Reply:
Good idea, thanks.